lib/gke-overrides.libsonnet

```diff
@@ -28,73 +42,208 @@ local ignore_dashboards = [
   'scheduler.json',
 ];
 
+// Define a mapping of alert/recordname to expression.
+// Overrides the expr field for the specifeid record name.
+local expr_overrides = {};
+
+// Define a mapping of alertname to label page:false
+// It will add a label `page:false` else it will add a `page:true`
+local page_false_critical = [
+  'AlertmanagerConfigInconsistent',
+  'ContainerCombinedIoHighOverTime',
+  'ClusterLowMemoryAvailable',
+  'ElasticsearchClusterHealthYELLOW',
+  'ElasticsearchHeapTooHigh',
+  'ElasticsearchLowDiskFree',
+  'HAProxyFrontendSessionUsage',
+  'HAProxyBackendResponseErrors',
+  'HAProxyBackendDown',
+  'HAProxyServerInBackendUpPercentageLow',
+  'KubePersistentVolumeInodeUsageCritical',
+  'KubePersistentVolumeFullInFourDays',
+  'PodOOMKilled',
+  'PrometheusBadConfig',
+  'PrometheusErrorSendingAlertsToAnyAlertmanager',
+  'PrometheusRuleFailures',
+  'PrometheusRemoteStorageFailures',
```

Comment:
PrometheusRemoteStorageFailures sounds scary - I suspect this could result in total prometheus outage? 

Do we know if this has ever happened?

Reply:
these are for REMOTE prometheus, we don't use them as of now .

We could actually just remove those rules

Reply:
Yeah, let's add them to the ignore/remove list (anything related to prometheus remotes)

Reply:
removed

  'PrometheusRuleFailures',
  'PrometheusRemoteStorageFailures',